.github/workflows/build_and_test.yml

```diff
@@ -31,7 +31,7 @@ jobs:
     - name: Build
       run: |
         . /opt/ros/foxy/setup.sh
-        colcon build --event-handlers console_cohesion+
+        colcon build --event-handlers console_cohesion+ --packages-up-to control_launch integration_launch localization_launch map_launch perception_launch planning_launch sensing_launch system_launch vehicle_launch
```

Comment:
Maybe we don't need this until all packages are ported?

Reply:
The problem is that if we don't filter the packages, the whole workspace will be built, including all the dependencies.

Reply:
But that would mean CI will always fail because it fails to find packages that are not ported.
I would suggest either update CI as we do the port, or always build whole workspace.

Reply:
Yes, that once a package is ported, move it from `--packages-ignore` to `--packages-up-to`. Building the entire workspace takes a lot of time now that Pilot.Auto has been added to the `build_depends.repos` file in this repository.